### ISSUE TITLE
Helm lint error 

### ISSUE THREAD

**Issue by alexnuttinck** [traefik/traefik-helm-chart#325]
Hello,

When I lint this helm chart, I got an error:

```
helm lint .

==> Linting .
[ERROR] templates/service.yaml: object name must be between 0 and 253 characters: ""

Error: 1 chart(s) linted, 1 chart(s) failed
```

helm version: 3.4.1

Thanks.

**Comment by jakubhajek** [comment]
Hello @alexnuttinck 

Would you please confirm what Helm version do you use? 

Seems that it has been fixed in Helm 3.4.1 by [#8862](https://github.com/helm/helm/pull/8862)

Thanks,

**Comment by alexnuttinck** [comment]
Hello @jakubhajek,

My current helm version:

```
helm version
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}
```
I confirm that it is fixed. It is now showed as a warning.

After upgrading to the last available helm version (3.5.1).

```
helm version
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
helm lint .
==> Linting .
[WARNING] templates/service.yaml: object name must be between 0 and 253 characters: ""

1 chart(s) linted, 0 chart(s) failed
```

Thanks @jakubhajek.
I close this issue.
